android/app/src/main/java/tuerantuer/app/integreat/MainActivity.java

```diff
@@ -3,17 +3,68 @@
 import android.content.res.Configuration;
 import android.os.Bundle;
 import com.facebook.react.ReactActivity;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
 
+import java.io.File;
 import java.util.Locale;
 
+
 public class MainActivity extends ReactActivity {
 
     private Locale currentLocale;
 
+    private void clearCachedContent()
+    {
+        // delete all files
+        File dir = getApplicationContext().getFilesDir();
+        if (dir.isDirectory())
```

Comment:
I think we can use Java 8 in the android project. If this is true then we can use this way of recursively deleting everything: https://www.baeldung.com/java-delete-directory#conclusion-1